admin_app/src/app/playground/components/PlaygroundComponents.tsx

```diff
@@ -118,7 +121,7 @@ const PersistentSearchBar = ({
                     marginTop: 2,
                     marginBottom: 2,
                     marginLeft: 0,
-                    minWidth: 160,
+                    width: 170,
```

Comment:
Fixed width to match longest menu option UrgencyDetection, so that changing the item doesn't change width!